docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,212 @@
+
+- Detects common Identity and Access Management (IAM) risks and help prioritizes misconfigurations across the public cloud to meet IAM compliance mandates- passwords, key rotation, and so on.
+
+- Facilitation of existing skillsets without time-consuming and costly training for managing compliance and security posture for cloud-native technologies, including containers, Kubernetes, and microservices.
+
+- Functions with IaaS, SaaS, and PaaS platforms in containerized, hybrid cloud, and multi-cloud environments.
+
+- Maintains an inventory of best practices for different cloud configurations and services.
+
+- Maps current configuration statuses to a security control framework or regulatory standard.
+
+- Monitors storage buckets, encryption, and account permissions for misconfigurations and compliance risks.
+
+- Support for native-cloud assets for multi-cloud deployment across AWS, Azure, Google, and Alibaba clouds using certified CIS/DISA STIG profiles to scan and detect security configuration issues.
+
+- Support SOC2 (Security Operation Center) compliance, voluntary compliance standard for service organizations, for cloud accounts.
+
+### Chef CSPM Framework
+
+The key features of the CSPM tools include the ability to:
+
+- End-to-end visibility governance
+  - Robust Detections: Easy to read, filterable scan reports (control summary) with detailed descriptions of all security controls and their results.
+  - Multi-Cloud Reporting: Cloud Integration support - GCP, AWS, Azure
+  - Data Aggregation: Data Export into 3rd Party tools (ServiceNow, Splunk)
+  - Azure: DevOps Governance and Policy Integration without Automate
+  - Third-party tools: Terraform, Postgres
+
+- Containers
+  - Best practices for Docker Enterprise Edition based on CIS
+
+- K8 Pods
+  - Security profiles for Kubernetes based on CIS benchmarks
+  - Remediation content for Kubernetes environment for security configuration violations based on CIS standards
+
+- Cloud
+  - Pre-built service and resource templates for custom policies across 100+ AWS, GCP, and Azure resources
+  - Policy Compliance for the latest OS Versions (CentOS, RHEL, Ubuntu) & DB versions (Oracle, PostgreSQL)  as per CIS best practices.
+
+- Host
+  - Remediation Content: Accessing Remediate profiles from Chef UI
+  - Fast, Scalable Remediation for security configuration violations based on CIS standards
+
+### Chef Products and AWS Shared Responsibility Model
+
+The **Chef Automate** can help you automate the ongoing compliance management and system state in your instances by providing a means to detect and correct security issues in a programmatic and repeatable fashion. You can configure on-demand or on-schedule scans, providing continuous and real-time insights into the security status across your environments. In addition, you can view the filterable dashboards to address and remediate any failures without slowing down your software development cycle.
```

Comment:
Extra "the" in these sentences. "**Chef Automate** can help you automate the ongoing compliance management and system state in your instances by providing a means to detect and correct security issues in a programmatic and repeatable fashion.